### ISSUE TITLE
clr-select drop down icon fails to meet min color contrast ratio 4:3:1 for accessibility

### ISSUE THREAD

**Issue by basalingayyakaramadi** [vmware/clarity#6509]
## Describe the bug

clr-select drop down icon fails to meet min color contrast ratio 4:3:1 for accessibility. even with normal flow and on on-hover.

## How to reproduce
https://clarity.design/angular-components/select/
you can verify in the clarity design page. you can use color contrast analyser tool to check the issue.

## Expected behavior

clr-select drop down icon color should meet min color contrast for default state/hover state.

## Versions

**Clarity project:**

- [ ] Clarity Core
- [x] Clarity Angular/UI

**Clarity version:**

- [ ] v3.x
- [ ] v4.x
- [x] v5.x

**Framework:**

- [x] Angular
- [ ] React
- [ ] Vue
- [ ] Other:

**Framework version:**
_ie: Angular 12_

**Device:**

- Type: [e.g. MacBook]

## Additional notes

_Add any other notes about the problem here._
<img width="1787" alt="select" src="https://user-images.githubusercontent.com/36897446/145188476-d8306930-38c7-4b92-ba63-73ab0aa6bea2.png">
![select-contrast](https://user-images.githubusercontent.com/36897446/145188488-ece6fbaf-121a-4655-8977-f37e78b03f74.png)

**Comment by gnomeontherun** [comment]
Is this related to internal testing? If so, we are tracking those things in JIRA with our accessibility team and don't want to duplicate issues.

@Shijir will your changes help here? This is an SVG so that might not be covered.
